mlflow/paddle/__init__.py

```diff
@@ -446,3 +448,104 @@ def predict(self, data):
 def _contains_pdparams(path):
     file_list = os.listdir(path)
     return any(".pdparams" in file for file in file_list)
+
+
+@experimental
+@autologging_integration(FLAVOR_NAME)
+def autolog(
+    log_every_n_epoch=1, log_models=True, disable=False, exclusive=False, silent=False,
+):  # pylint: disable=unused-argument
+    """
+    Enables (or disables) and configures autologging from `PaddlePaddle
+    <https://pytorch-lightning.readthedocs.io/en/latest>`_ to MLflow.
+
+    Autologging is performed when you call the `fit` method of `paddle.Model`_.
+
+    .. _paddle.Model:
+        https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/Model_en.html
+
+    Explore the complete `PaddlePaddle MNIST \
+    <https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/Model_en.html#model>`_ for
+    an expansive example.
```

Comment:
Can we apply https://github.com/mlflow/mlflow/pull/4751#discussion_r698615100 to other links?

Reply:
We also have this link related to MNIST, but it currently only supports in Chinese. `https://www.paddlepaddle.org.cn/documentation/docs/zh/tutorial/cv_case/image_classification/image_classification.html#lenetmnist`. What do you think? It seems that most of the model training examples are in Chinese only.

Reply:
Done~